src/t8_forest/t8_forest_vtk.cxx

```diff
@@ -428,8 +427,6 @@ t8_forest_to_vtkUnstructuredGrid (t8_forest_t forest,
   vtkDoubleArray    **dataArrays;
   dataArrays = T8_ALLOC (vtkDoubleArray *, num_data);
 
-  t8_cmesh_t          cmesh = t8_forest_get_cmesh (forest);
```

Comment:
Can you also remove the two includes then?
```
#include <t8_cmesh.h>
[...]
#include "t8_cmesh/t8_cmesh_trees.h"
```

Reply:
Done!